cmd/common-main_test.go

```diff
@@ -182,3 +182,57 @@ MINIO_ROOT_PASSWORD=minio123`,
 		})
 	}
 }
+
+func Test_subPathFromRedirectURL(t *testing.T) {
+	type args struct {
+		redirectURL string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "no sub-path",
```

Comment:
This test is not needed and the function doesn't exist anymore.

Reply:
done